src/test/java/expresslibrary/logic/commands/BorrowCommandTest.java

```diff
@@ -0,0 +1,126 @@
+package expresslibrary.logic.commands;
+
+import expresslibrary.commons.core.Messages;
+import expresslibrary.commons.core.index.Index;
+import expresslibrary.model.Model;
+import expresslibrary.model.ModelManager;
+import expresslibrary.model.UserPrefs;
+import expresslibrary.model.book.Book;
+import expresslibrary.model.person.Person;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static expresslibrary.logic.commands.CommandTestUtil.*;
+import static expresslibrary.logic.commands.CommandTestUtil.assertCommandFailure;
+import static expresslibrary.model.Model.PREDICATE_SHOW_ALL_BOOKS;
+import static expresslibrary.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static expresslibrary.testutil.TypicalExpressLibrary.getTypicalExpressLibrary;
+import static expresslibrary.testutil.TypicalIndexes.INDEX_FIRST;
+import static expresslibrary.testutil.TypicalIndexes.INDEX_SECOND;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code BorrowCommand}.
+ */
+public class BorrowCommandTest {
+
+    private Model model = new ModelManager(getTypicalExpressLibrary(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        Person personToLend = model.getFilteredPersonList().get(INDEX_FIRST.getZeroBased());
+        Book bookToBorrow = model.getFilteredBookList().get(INDEX_SECOND.getZeroBased());
+
+        LocalDate dueDate = LocalDate.now().plusDays(1);
+        BorrowCommand borrowCommand = new BorrowCommand(INDEX_FIRST, INDEX_SECOND, dueDate);
+
```

Comment:
Just add this and the test case should pass.
```suggestion
personToLend.borrowBook(bookToBorrow);
```